src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -134,6 +134,12 @@ export default class CancerStudySelector extends React.Component<ICancerStudySel
         );
     });
 
+    @computed
```

Comment:
seems like this belongs in the store

Reply:
agreed. also please rename to `onlyOneReferenceGenome` or something like that, to agree with the getter syntax.